grug/testing/tests/cron.rs

```diff
@@ -36,6 +41,18 @@ mod tester {
 
         Ok(Response::new().add_message(Message::transfer(job.receiver, job.coin).unwrap()))
     }
+
+    pub fn failing_instantiate(_: MutableCtx, _: Empty) -> StdResult<Response> {
+        Ok(Response::new())
+    }
+
+    pub fn failing_cron_execute(ctx: SudoCtx) -> StdResult<Response> {
+        SHOULD_BE_EMPTY.save(ctx.storage, &()).unwrap();
+
+        Err(StdError::host(
+            "This cronjob is supposed to fail".to_string(),
+        ))
```

Comment:
`StdError::host` means the host returned an error, so this is a misuse here. you can just use `anyhow::Result` for this function.